_examples/logging/main.go

```diff
@@ -13,30 +13,28 @@ import (
 	"fmt"
 	"net/http"
 	"os"
+	"reflect"
 	"time"
 
-	"golang.org/x/exp/slog"
+	"log/slog"
```

Comment:
This won't compile on Go 1.20 and below. Should we add a compiler directive for the file?